libraries/management/commands/import_library_versions.py

```diff
@@ -0,0 +1,137 @@
+            )
+            continue
+
+        gitmodules = parser.parse_gitmodules(raw_gitmodules.decode("utf-8"))
+
+        for gitmodule in gitmodules:
+            click.echo(f"Processing module {gitmodule['module']}...")
+            if gitmodule["module"] in updater.skip_modules:
+                click.echo(f"Skipping module {gitmodule['module']}.")
+                continue
+
+            libraries_json = client.get_libraries_json(repo_slug=gitmodule["module"])
+
+            # If the libraries.json file exists, we can use it to get the library info
+            if libraries_json:
+                libraries = []
+                if type(libraries_json) is list:
+                    for library in libraries_json:
+                        data = parser.parse_libraries_json(library)
+                        libraries.append(data)
+                else:
+                    libraries.append(parser.parse_libraries_json(libraries_json))
+
+                for lib_data in libraries:
+                    try:
+                        library = Library.objects.get(name=lib_data["name"])
+                    except Library.DoesNotExist:
+                        click.echo(
+                            f"Could not find library by gitmodule name; skipping {gitmodule['module']}"
+                        )
+                        skipped.append(
+                            {
+                                "version": version.name,
+                                "library": gitmodule["module"],
+                                "reason": "Could not find library by gitmodule name",
+                            }
+                        )
+                    else:
+                        library_version, _ = LibraryVersion.objects.get_or_create(
+                            version=version, library=library
+                        )
+                        click.echo(
+                            f"Saved library version {library_version}. Created? {_}"
+                        )
+                        updater.update_maintainers(
+                            library_version, maintainers=lib_data["maintainers"]
+                        )
+                        click.secho(
+                            f"Updated maintainers for {library_version}.", fg="green"
+                        )
+
+                    click.echo(f"Saved library version {library_version}.")
+            else:
+                # This can happen with older tags; the libraries.json file didn't always exist, so
+                # when it isn't present, we search for the library by the module name and try to save
+                # the LibraryVersion that way.
+                click.echo(
+                    f"Could not get libraries.json for {gitmodule['module']}; will try to save by gitmodule name."
+                )
+                try:
+                    library = Library.objects.get(name=gitmodule["module"])
```

Comment:
This would also be simplified using `name` defined above, if we choose to do that:
```suggestion
                    library = Library.objects.get(name=name)
```